.gitleaks.toml

```
@@ -5,4 +5,7 @@ useDefault = true
 [allowlist]
 # skip base64 encoded images, which might have substrings that look like tokens
 regexTarget = "line"
-regexes = ['''(?i)<img src="data:image\/.+;base64,.+?".+\/>''']
+regexes = [
+    '''(?i)<img src="data:image\/.+;base64,.+?".+\/>''',
+    '''i--;break;case V\.ui\.keyCode\.END:.+;break;case V\.ui\.keyC'''
```

This line is the jQuery inclusion (which probably has changed between versions of rmarkdown, hence a new error), so to be a bit more general/robust to future changes that could look like secrets, we might want to match the start of the line with something like:

```suggestion
    # skip jQuery definition function
    '''^!function\(.+jQuery'''
```

I think the comment should be okay within the array, so I would move the comment on line 6 to preface the first regex.